exchanges/bithumb/bithumb_wrapper.go

```diff
@@ -527,15 +527,16 @@ func (b *Bithumb) ModifyOrder(ctx context.Context, action *order.Modify) (*order
 		return nil, err
 	}
 
-	return &order.Modify{
-		Exchange:  action.Exchange,
-		AssetType: action.AssetType,
-		Pair:      action.Pair,
-		ID:        o.Data[0].ContID,
-		Price:     float64(int64(action.Price)),
-		Amount:    action.Amount,
-		Side:      action.Side,
-	}, nil
+	resp, err := action.DeriveModifyResponse()
```

Comment:
Unrelated, but I can't even find documentation on the ability to modify an order

Reply:
I have no keys to test this either, I am thinking about removing this functionality.